chunky/src/java/se/llbit/chunky/world/EmptyChunk.java

```diff
@@ -46,16 +45,13 @@ private EmptyChunk() {
     surface = IconLayer.CORRUPT;
   }
 
-  @Override public synchronized void getBlockData(int[] blocks, byte[] biomes,
-      Collection<CompoundTag> tileEntities, Collection<CompoundTag> entities,
-      BlockPalette palette) {
-    for (int i = 0; i < X_MAX * Y_MAX * Z_MAX; ++i) {
-      blocks[i] = 0;
-    }
-
-    for (int i = 0; i < X_MAX * Z_MAX; ++i) {
-      biomes[i] = 0;
+  @Override public synchronized ChunkData getChunkData(ChunkData chunkData, BlockPalette palette) {
```

Comment:
Can you rename `chunkData` to `reuseChunkData` (in the interface and all implementations)? I know the javadoc says it, but I still was confused for the second time now.